controller.go

```diff
@@ -94,6 +94,9 @@ type NetworkController interface {
 	// Sandboxes returns the list of Sandbox(s) managed by this controller.
 	Sandboxes() []Sandbox
 
+	// RenameSandbox changes the sandbox name; should be called on a container rename
+	RenameSandbox(containerID string, options ...SandboxOption) error
+
```

Comment:
Also, rename is a very specific API. I don't think we need SandboxOptions for that.
I think `Sandbox.Rename(newName string)` is enough.